example/src/components/Checkout/components/StoredPaymentMethodsList.tsx

```diff
@@ -0,0 +1,77 @@
+import type { StoredPaymentMethod } from '@adyen/react-native';
+import { useMemo } from 'react';
+import { View } from 'react-native';
+import type { StoredCardPaymentMethod } from '../../../api/types';
+import AdaptiveText from '../../common/AdaptiveText';
+import PaymentMethodListItem from './PaymentMethodListItem';
+import Styles from '../../common/Styles';
+
+interface StoredPaymentMethodsListProps {
+  storedPaymentMethods: StoredPaymentMethod[] | undefined;
+  makePayment: (storedCard: StoredCardPaymentMethod) => void;
+}
+
+const StoredPaymentMethodsList = ({
+  storedPaymentMethods,
+  makePayment,
+}: StoredPaymentMethodsListProps) => {
+  const storedCards = useMemo(() => {
```

Comment:
Does mobile only have stored cards?

Reply:
No, there is a few more (Cash App, Plaid, PayPal) but I haven't check compatibility yet.